src/Altinn.App.Core/Internal/Process/ProcessTasks/ServiceTasks/EFormidlingServiceTask.cs

```diff
@@ -0,0 +1,77 @@
+using Altinn.App.Core.Configuration;
+using Altinn.App.Core.EFormidling.Interface;
+using Altinn.Platform.Storage.Interface.Models;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.App.Core.Internal.Process.ProcessTasks.ServiceTasks;
+
+internal interface IEFormidlingServiceTask : IServiceTask { }
+
+/// <summary>
+/// Service task that sends eFormidling shipment, if EFormidling is enabled in config.
+/// </summary>
+public class EFormidlingServiceTask : IEFormidlingServiceTask
+{
+    private readonly ILogger<EFormidlingServiceTask> _logger;
+    private readonly IEFormidlingService? _eFormidlingService;
+    private readonly IOptions<AppSettings>? _appSettings;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EFormidlingServiceTask"/> class.
+    /// </summary>
+    public EFormidlingServiceTask(
+        ILogger<EFormidlingServiceTask> logger,
+        IEFormidlingService? eFormidlingService = null,
+        IOptions<AppSettings>? appSettings = null
+    )
+    {
+        _logger = logger;
+        _eFormidlingService = eFormidlingService;
+        _appSettings = appSettings;
+    }
+
+    /// <inheritdoc />
+    public string Type => "eFormidling";
+
+    /// <inheritdoc/>
+    public async Task Execute(string taskId, Instance instance)
+    {
+        if (_appSettings?.Value.EnableEFormidling is false)
+        {
+            _logger.LogWarning(
+                "EFormidling is not enabled in appsettings.json. No eFormidling shipment will be sent, but the service task will be completed."
```

Comment:
Does this message mean that I have tried to enable the eformidling service task in BPMN without enabling it? Could that be explained in a better way?